[quantization] Changing observer name

z-a-f opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#27779 [quantization] Changing observer name**
* #27670 [quantization] Refactoring names for consistency

Differential Revision: [D17886605](https://our.internmc.facebook.com/intern/diff/D17886605)

Commit:
[quantization] Changing observer name

zafartahirov commented:
@jerryzh168 I think I overdid that  :/

Commit:
Update on "[quantization] Changing observer name"


Differential Revision: [D17886605](https://our.internmc.facebook.com/intern/diff/D17886605)

jerryzh168 commented on torch/nn/quantized/modules/conv.py:
```diff
@@ -222,20 +222,20 @@ def from_float(cls, mod):
+            assert hasattr(mod, 'activation_post_process'), 'Input QAT module must have observer attached'
             weight_observer = mod.weight_fake_quant
-            activation_observer = mod.observer
+            activation_observer = mod.activation_post_process
```
please change `activation_observer` to `activation_post_process` here as well

jerryzh168 commented on torch/nn/quantized/modules/conv.py:
```diff
@@ -222,20 +222,20 @@ def from_float(cls, mod):
                                          mod.running_var, mod.eps, mod.gamma, mod.beta)
-            assert hasattr(mod, 'observer'), 'Input QAT module must have observer attached'
+            assert hasattr(mod, 'activation_post_process'), 'Input QAT module must have observer attached'
             weight_observer = mod.weight_fake_quant
```
and `weight_observer` to `weight_post_process`?

jerryzh168 commented:
> @jerryzh168 I think I overdid that :/

what do you mean?

z-a-f commented:
> > @jerryzh168 I think I overdid that :/
> 
> what do you mean?

I mean too many renames :)

Commit:
Update on "[quantization] Changing observer name"


Differential Revision: [D17886605](https://our.internmc.facebook.com/intern/diff/D17886605)

Commit:
Update on "[quantization] Changing observer name"


Differential Revision: [D17886605](https://our.internmc.facebook.com/intern/diff/D17886605)

jerryzh168 reviewed (approved):
LG
